panels/user-accounts/um-password-dialog.c

```diff
@@ -566,6 +625,18 @@ um_password_dialog_new (void)
 
         g_object_unref (builder);
 
+        um->cancellable = g_cancellable_new ();
```

Comment:
It may not matter if you do the sync creation, but otherwise this should be cancelled on finalize (and unreffed, otherwise it gets leaked)